### ISSUE TITLE
Add SC1091 in the readme

### ISSUE THREAD

**Issue by Hongbo-Miao** [hadolint/hadolint#905]
- [ ] This is a bug report
- [x] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

I got

> SC1091 info: Not following: File not included in mock.

for the first line of

```Dockerfile
RUN . /venv/bin/activate && \
  poetry install --no-dev --no-root
```

However, I don't understand what that means and how to fix.

It would be great to add info about SC1091 in the readme, thanks! 😃

### Actual behavior

SC1091 is currently missing in the readme.

**Comment by lorenzo** [comment]
I think SC1091 should be disabled. It was added recently by shellcheck, but it makes no sense for hadolint

**Comment by Hongbo-Miao** [comment]
I see, thanks for clarifying, @lorenzo !
BTW, for people who are interersted in, here is the SC1091 from `shellcheck` https://www.shellcheck.net/wiki/SC1091

**Comment by softwaredevelop** [comment]
This is a shellchek problem. Use quotation mark.
```Dockerfile
RUN \
    /bin/bash -c \
    "source $HOME/.sdkman/bin/sdkman-init.sh; \
    sdk install java 17.0.2-ms; \
    sdk install gradle 7.4.2; \
    rm -rf $HOME/.sdkman/archives/*; \
    rm -rf $HOME/.sdkman/tmp/*"
```

**Comment by Hongbo-Miao** [comment]
Hi @softwaredevelop thanks for sharing, however, I don't think that is not a right way to fix though. https://www.shellcheck.net/wiki/SC1091 has more info.

**Comment by softwaredevelop** [comment]
Yes. But I use .sh files too and shellcheck use for these.

For complex bash commands in Dockerfile:
```Dockerfile
COPY library-scripts/* /tmp/scripts/

RUN \
    bash /tmp/scripts/common-debian.sh
```
Dockerfile languge is not a perfect language. As you feel 😉

**Comment by aminya** [comment]
Why shellcheck is not able to find the file while it exists on the system? I'd prefer shellcheck to check things for me.
